src/fetchtastic/setup_config.py

```
@@ -1482,6 +1463,50 @@ def _setup_firmware(
+            parsed_current = DEFAULT_FIRMWARE_NIGHTLY_VERSIONS_TO_KEEP
+        nightly_input = _safe_input(
+            f"\nHow many firmware nightly builds would you like to keep? (current: {parsed_current}): ",
+            default=str(parsed_current),
+        ).strip()
+        parsed_keep = _parse_non_negative_int(nightly_input)
+        if parsed_keep is not None and parsed_keep >= 1:
+            config["FIRMWARE_NIGHTLY_VERSIONS_TO_KEEP"] = parsed_keep
+        elif not nightly_input:
+            config["FIRMWARE_NIGHTLY_VERSIONS_TO_KEEP"] = parsed_current
+        else:
+            print("Invalid value — keeping current value. " "Minimum is 1.")
```

Ruff (0.15.20)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)